### PR TITLE
Feature/adding user guide

### PR DESCRIPTION
**Pull request by Javierverbel**
### Description
- (purpose of the PR, how did you implement it)

### Review process
- (explain how to review it)

### Pre-approval checklist
- [ ] The code builds clean without any errors or warnings
- [ ] I've added/updated tests
- [ ] I've added/updated documentation if needed

### CODE REVIEW

**Review by franjpr — APPROVED** [review]

**Review by Memphisd — APPROVED** [review]
made some rearangements and some changes, Javi should take a look before merging

**Pull request merged**
